react/react-hooks/why-are-react-hooks-required.md

```diff
@@ -0,0 +1,232 @@
+# Why Are React Hooks Required?
+
+If you are new to Web Development or React in particular, you might have heard this buzz word "React Hooks" a lot and might be finding it overwhelming too. You may be having questions like – "Should I learn it? Is it important? Is it even required for a beginner like me?".
+
+While I cannot straight away answer your questions, what I can do is help you understand why do React Hooks exist and what motivated their intoduction to React by the React Team, so that you can decide for yourself whether to learn React Hooks or not.
```

Comment:
Is it possible to break this sentence into smaller sentences? Just a suggestion, not necessary